llvm/include/llvm/IR/RuntimeLibcalls.h

```diff
@@ -68,10 +68,7 @@ struct RuntimeLibcallsInfo {
     return LibcallCallingConvs[Call];
   }
 
-  iterator_range<const char **> getLibcallNames() {
-    return llvm::make_range(LibcallRoutineNames,
-                            LibcallRoutineNames + RTLIB::UNKNOWN_LIBCALL);
-  }
+  ArrayRef<const char *> getLibcallNames() const { return LibcallRoutineNames; }
```

Comment:
If I'm reading this right, this is now going to include the entry for UNKNOWN_LIBCALL as well. Is that intentional?